scripts/forest_fires_portugal.json

```diff
@@ -1,5 +1,5 @@
 {
-    "citation": "[Cortez and Morais, 2007] P. Cortez and A. Morais. A Data Mining Approach to Predict Forest Fires using Meteorological Data. In J. Neves, M. F. Santos and J. Machado Eds., New Trends in Artificial Intelligence, Proceedings of the 13th EPIA 2007 - Portuguese Conference on Artificial Intelligence, December, Guimar\u00e3es, Portugal, pp. 512-523, 2007. APPIA, ISBN-13 978-989-95618-0-9.",
+    "citation": "Cortez and Morais, 2007] P. Cortez and A. Morais. A Data Mining Approach to Predict Forest Fires using Meteorological Data. In J. Neves, M. F. Santos and J. Machado Eds., New Trends in Artificial Intelligence, Proceedings of the 13th EPIA 2007 - Portuguese Conference on Artificial Intelligence, December, Guimarães, Portugal, pp. 512-523, 2007. APPIA, ISBN-13 978-989-95618-0-9.",
```

Comment:
How about we just make the beginning "Cortez, P. and A. Morais. 2007. A Data Mining..."